### DISCUSSION TITLE
Blank Page on Load

### DISCUSSION THREAD

**Discussion by Bovive** [karlomikus/bar-assistant#100] [Q&A] [answered]
Hello,

I just installed and I am receiving a blank page when accessing Bar Assistant. The favicon is loading the background is colored. There is however no other content. I am fairly certain I am doing something wrong. I cannot find any logs other than this when accessing the page:

```
2023-04-25 13:47:11 172.25.0.1 - - [25/Apr/2023:18:47:11 +0000] "GET /config.js HTTP/1.1" 304 0 "https://localbar.mysite.com/bar" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36 Edg/112.0.1722.58" "172.23.0.1"
2023-04-25 13:47:13 172.25.0.1 - - [25/Apr/2023:18:47:13 +0000] "GET /sw.js HTTP/1.1" 304 0 "https://localbar.mysite.com/sw.js" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36 Edg/112.0.1722.58" "172.23.0.1"
```

Everything is running in Docker and the logs indicated "Application Ready". If anyone would have a suggestion to point me in the right direction I would greatly appreciate it. Thank you!

**Comment by karlomikus** [comment]
Hi, this looks like some issue with salt-rim.

Check your browser console for errors.

Try hard refresh (ctrl+f5 on firefox). You could also try on another browser to see if it's browser specific.

**Comment by Bovive** [comment]
Thanks for the tips. I did try Firefox as you suggested on a different computer but experienced the same thing. I checked the console and this is what I am seeing:

<img width="848" alt="error" src="https://user-images.githubusercontent.com/4776549/234382857-34f91194-9a98-460b-a68f-f79c07fadf1d.PNG">

**Comment by karlomikus** [comment, marked as answer]
Looks like issue with env variable.

Can you post what you have as `DEFAULT_LOCALE=` in your docker compose file.

You could also remove that variable, restart the containers and see if that fixes it

**Reply by Bovive** [reply]
Sure thing. I have it the same as was in the example compose file:

`DEFAULT_LOCALE="en-US"`

**Reply by Bovive** [reply]
Deleting it all and trying one more time after removing the DEFAULT_LOCALE from the docker-compose file did the trick. Thanks for the help!

**Comment by Bovive** [comment]
Forgot to mention, I tried deleting the "DEFAULT_LOCALE=" env variable altogether, but it did not work either.

**Comment by karlomikus** [comment]
Can you post what is the output of this command:
```
docker compose exec salt-rim cat /var/www/html/config.js
```
